templates/_layouts/_root.html

```diff
@@ -14,7 +14,9 @@
 
     <title>{% block title %}{{ title }}{% endblock %}{% if self.title() %} | {% endif %}Vanilla documentation</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <link rel="icon" href="https://assets.ubuntu.com/v1/ab36e6ed-vanilla_favicon_32px.png" type="image/x-icon" />
+    <link rel="icon" type="image/x-icon" sizes="48x48" href="https://assets.ubuntu.com/v1/f1464d22-vanilla_favicon_48px.png"  />
```

Comment:
is this correct to have `type="image/x-icon"` when it's a png, not an .ico?

Reply:
@bartaz `image/png` is technically correct - will adjust